suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -72,36 +97,9 @@ export const TransactionList = ({
             })),
         [accountTransactionsByMonth, transactionMonthKeys],
     );
-    interface RenderItemParams {
-        item: WalletAccountTransaction;
-        section: typeof sectionsData[number];
-        index: number;
-    }
-
-    const renderItem = useCallback(
-        ({ item, section, index }: RenderItemParams) => (
-            <TransactionListItem
-                key={item.txid}
-                transaction={item}
-                isFirst={index === 0}
-                isLast={index === section.data.length - 1}
-            />
-        ),
-        [],
-    );
 
-    interface RenderSectionHeaderParams {
-        section: {
-            monthKey: string;
-        };
-    }
-
-    const renderSectionHeader = useCallback(
-        ({ section: { monthKey } }: RenderSectionHeaderParams) => (
-            <TransactionListGroupTitle key={monthKey} monthKey={monthKey} />
-        ),
-        [],
-    );
+    const renderItemCallback = useCallback(renderItem, []);
```

Comment:
Both of this `useCallback` are not necessary here. `useCallback` is used when you don't want to create new function again and again with every rerender, but since you placed these before component, they will created just once, so you can use them directly.

Reply:
fixed in [c284988](https://github.com/trezor/trezor-suite/pull/7392/commits/c2849888ab32eccc8356c1d8981781f2b37cab60)